scripts/jenkins-yoctobuild-build.sh

```diff
@@ -552,6 +552,11 @@ build_and_test_client() {
                 sudo patch -p1 /usr/local/lib/python2.7/dist-packages/fabric/network.py b60247d78e9a7b541b3ed5de290fdeef2039c6df.patch || true
             fi
 
+	    # Zeus and older do not have this.
+	    if grep -q mender-testing-enabled $WORKSPACE/meta-mender/meta-mender-core/classes/mender-setup.bbclass; then
+		echo 'MENDER_FEATURES_ENABLE_append = " mender-testing-enabled"' >> $BUILDDIR/conf/local.conf
+	    fi
```

Comment:
This is wrong, as the features are now listed in `mender-maybe-setup.bbclass`.

```suggestion
	    if [[ -f $WORKSPACE/meta-mender/meta-mender-core/classes/mender-maybe-setup.bbclass ]]; then
	        if grep -q mender-testing-enabled $WORKSPACE/meta-mender/meta-mender-core/classes/mender-maybe-setup.bbclass; then
		    echo 'MENDER_FEATURES_ENABLE_append = " mender-testing-enabled"' >> $BUILDDIR/conf/local.conf
	        fi
	    fi
```

I believe this should fix all the test failures that we have seen in https://github.com/mendersoftware/meta-mender/pull/1084; at least for `qemux86_64_uefi_grub` all test feilures where related to accessing `/data/etc/...`

Reply:
Aha! Good catch! I'll update it ASAP.

Reply:
Btw, is the `-f` test necessary? It's implicit in the `grep` isn't it?

Reply:
Yes, good point. The `grep` would fail if the file doesn't exist (which is exactly what was happening now).